java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -52,4 +53,24 @@ public interface StreamBaseCommands {
      * }</pre>
      */
     CompletableFuture<String> xadd(String key, Map<String, String> values, StreamAddOptions options);
+
+    /**
+     * Trims the stream by evicting older entries.
+     *
+     * @see <a href="https://redis.io/commands/xtrim/">redis.io</a> for details.
+     * @param key The key of the stream.
+     * @param options Stream trim options.
+     * @return The number of entries deleted from the stream.
+     * @example
+     *     <pre>{@code
+     * // A nearly exact trimming of the stream to at least a length of 10, limited to 10 entries
```

Comment:
Don't quite understand what you mean by the last part. "least a length of 10, limited to 10 entries" why is least a length of 10 not enough?